.github/workflows/CI_build_libs.yml

```diff
@@ -0,0 +1,122 @@
+name: CI_build_libs
+
+on: [push, pull_request]
+
+jobs:
+  build_windows:
+
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        build_configuration: [Release]
+        build_platform: ["NMake Makefiles"]
+
+    steps:
+
+    - name: Checkout repo
+      uses: actions/checkout@v4
+
+    - name: Add nmake
+      uses: ilammy/msvc-dev-cmd@v1
+
+    - name: generate cmake
+      run: |
+           cmake -G "${{ matrix.build_platform }}" -DCMAKE_BUILD_TYPE="${{ matrix.build_configuration }}" -B c:\_build  D:\a\AusweisApp\AusweisApp\libs
+
+    - name: build cmake
+      run: |
+           cmake --build c:\_build --config ${{ matrix.build_configuration }}
+
+  build_linux:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        build_configuration: [Release]
+        build_platform: ["Ninja"]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install packages via apt
+      run: |
+           sudo apt-get update -qq && sudo apt install -y cmake pkg-config libssl-dev libudev-dev libhttp-parser-dev libpcsclite-dev libgl1-mesa-dev libdbus-1-dev libclang-15-dev libclang-13-dev ninja-build
```

Comment:
why libclang15 and 13?

Reply:
See https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md. Clang 13, 14 and 15 is available and it seems it is not always stable which one is found.
Needed to avoid issue:

> CMake Warning (dev) at /home/runner/work/AusweisApp/AusweisApp/_build/b/src/qt-build/qtbase/lib/cmake/Qt6Qml/Qt6QmlBuildInternals.cmake:404 (message):
>   INSTALL_SOURCE_QMLTYPES option is deprecated and should not be used.
>   Please port your module to use declarative type registration.
> Call Stack (most recent call first):
>   qtwebsockets/src/imports/qmlwebsockets/CMakeLists.txt:4 (qt_internal_add_qml_module)
> This warning is for project developers.  Use -Wno-dev to suppress it.
> 
> -- Configuring submodule 'qtscxml'
> -- Configuring submodule 'qtactiveqt'
> -- Configuring submodule 'qttools'
> CMake Error at /usr/lib/llvm-14/lib/cmake/clang/ClangTargets.cmake:750 (message):
>   The imported target "clangBasic" references the file
> 
>      "/usr/lib/llvm-14/lib/libclangBasic.a"
> 
>   but this file does not exist.  Possible reasons include:
> 
>   * The file was deleted, renamed, or moved to another location.
> 
>   * An install or uninstall procedure did not complete successfully.
> 
>   * The installation package was faulty and contained
> 
>      "/usr/lib/llvm-14/lib/cmake/clang/ClangTargets.cmake"
> 
>   but not all the files it references.
> 
> Call Stack (most recent call first):
>   /usr/lib/cmake/clang-14/ClangConfig.cmake:19 (include)
>   qttools/cmake/FindWrapLibClang.cmake:17 (find_package)
>   qtbase/cmake/QtFindPackageHelpers.cmake:147 (find_package)
>   qttools/configure.cmake:20 (qt_find_package)
>   qtbase/cmake/QtFeature.cmake:592 (include)
>   qttools/src/CMakeLists.txt:18 (qt_feature_evaluate_features)
> 
> 
> -- Configuring incomplete, errors occurred!
> 
>